google/ground-android
# [Offline sync] Allow user to specify connection type for sync

Fixes #408

@dturner PTAL

Tested manually:

 - Turn the setting to sync `Over Wi-Fi only` on
 - Toggle Wifi off and keep Mobile Data on
 - Add a new observation and also attach a photo
 - Click "save"
 - Observe that the form data is saved but the photo isn't uploaded
 - Turn on wifi
 - Photo gets synced in the background automatically
